Resources/config/orm.xml

```diff
@@ -213,5 +213,18 @@
 
             <tag name="console.command" command="doctrine:mapping:import" />
         </service>
+
+        <service id="doctrine.ulid_generator" class="Symfony\Bridge\Doctrine\IdGenerator\UlidGenerator" public="false">
+            <argument type="service" id="doctrine.ulid_factory" on-invalid="ignore" />
```

Comment:
I'm confused, there is no constructor in UlidGenerator, so why are we defining arguments? Same for UuidGenerator

Reply:
I'm removing this and will submit as a separate PR, when the corresponding factories are merged into Symfony.